languages/source.nim

```diff
@@ -775,10 +802,20 @@ const lang* = language:
       let val_2 = val_1[n_1]
       conclusion At(array(*val_1), IntVal(n_1)), val_2
 
+    rule "E-at-tuple":
+      condition 0 <= n_1
```

Comment:
Perhaps it's just me, but I still find this "backwards", and need to mentally flip this to `n_1 >= 0` in order to understand it.

Soft comment, doesn't need to change, just wondering if you trip up on this too?

Reply:
> Soft comment, doesn't need to change, just wondering if you trip up on this too?

Yea, I still trip on it sometimes, though I'm getting somewhat used to this notation, as "upper and lower bound for `x`" is often written `lb <= x <= ub` in the literature.

The only reason I'm using the "constant before variable" notation here is because the meta-language has no `>=` function :D. There's no reason for the meta-language to not have it -- the current notation just didn't bother me enough to PR `>` and `>=`.